HLTrigger/HLTfilters/plugins/plugins.cc

```diff
@@ -130,6 +130,8 @@ typedef HLTDoublet<RecoEcalCandidate, PFMET> HLT2PhotonPFMET;
 #include "HLTDoubletSinglet.h"
 typedef HLTDoubletSinglet<PFTau, PFTau, PFJet> HLT3DoublePFTauPFJet;
 typedef HLTDoubletSinglet<RecoChargedCandidate, PFTau, PFJet> HLT3MuonPFTauPFJet;
+typedef HLTDoubletSinglet<PFJet, PFJet, PFJet> HLT3TriplePFJet;
+typedef HLTDoubletSinglet<RecoChargedCandidate, PFJet, PFJet> HLT3MuonDoublePFJet;
```

Comment:
Given that:
```
 *  The filter itself compares only objects from collection 3
 *  with objects from collections 1 and 2.
 ```
are you sure of the above or should it be
```
typedef HLTDoubletSinglet<PFJet, PFJet, RecoChargedCandidate> HLT3DoublePFJetMuon;
```
???

Reply:
That filter will be used in the monitoring path, where we first select one muon and one tau (reconstructed as a jet as we are going to use PNet) and at the end we select the _true_ jet, which shouldn't match the muon and the tau. So I think it is ok how it is right now.